### ISSUE TITLE
richdocuments acceptance tests fail after core change

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#36090]
`richdocuments` acceptance tests that check:
```
Then the additional sharing attributes for the response should include
```

have been failing since core PR #36024 commit https://github.com/owncloud/core/pull/36024/commits/2ef190a4fce0d0ab6173da1304ffaa35e95ede95

It seems that the attributes are not getting reported, and maybe not getting set.

e.g. https://drone.owncloud.com/owncloud/richdocuments/419/10/9

```
make test-acceptance-webui BEHAT_FEATURE=tests/acceptance/features/webUISecureView/main.feature:51
composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
BEHAT_BIN=vendor-bin/behat/vendor/bin/behat ../../tests/acceptance/run.sh --remote --type webUI
Script path: /home/phil/git/owncloud/core/tests/acceptance
Not using php inbuilt server for running scenario ...
Updating .htaccess for proper rewrites
Could not reliably determine if 'mod_rewrite' module is enabled on the local server, because the server does not seem to be running behind Apache.
"maxDuration":"3600"
{"extensions" : {"Behat\\MinkExtension" : {"browser_name": "chrome", "base_url" : "http://172.17.0.1:8080", "selenium2":{"capabilities": {"marionette":null, "browser": "chrome", "name": "ownCloud non-CI", "extra_capabilities": {"maxDuration":"3600"}}, "wd_host":"http://localhost:4445/wd/hub"}}, "SensioLabs\\Behat\\PageObjectExtension" : {}}}
Running webUISecureView tests tagged ~@skipWhenTestingRemoteSystems&&~@skipOnOcV10&&~@skipOnOcV10.3&&~@skipOnOcV10.3.0&&@webUI&&~@skip on browser 'chrome' 
@webUI @insulated @disablePreviews
Feature: Secure View
  As an admin
  I want to be able to enable secure view
  So that users can share files with very restricted access to documents

  Scenario: Admin enables secure view, user shares without edit permissions with default secure view permissions set                                   # /home/phil/git/owncloud/core/apps-external/richdocuments/tests/acceptance/features/webUISecureView/main.feature:51
    Given the administrator has added config key "secure_view_option" with value "true" in app "richdocuments"                                         # OccContext::theAdministratorAddsConfigKeyWithValueInAppUsingTheOccCommand()
    And user "user0" has been created with default attributes and skeleton files                                                                       # FeatureContext::userHasBeenCreatedWithDefaultAttributes()
    And user "user1" has been created with default attributes and without skeleton files                                                               # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()
    And user "user0" has logged in using the webUI                                                                                                     # WebUILoginContext::theUserLogsInUsingTheWebUI()
    When the user shares folder "simple-folder" with user "User One" using the webUI                                                                   # WebUISharingContext::theUserSharesFileFolderWithUserUsingTheWebUI()
    And the user sets the sharing permissions of user "User One" for "simple-folder" using the webUI to                                                # WebUISharingContext::theUserSetsTheSharingPermissionsOfForOnTheWebUI()
      | edit | no |
    And user "user0" gets the info of the last share using the sharing API                                                                             # FeatureContext::userGetsInfoOfLastShareUsingTheSharingApi()
    Then the fields of the last response should include                                                                                                # FeatureContext::checkFields()
      | id                     | A_NUMBER       |
      | item_type              | folder         |
      | item_source            | A_NUMBER       |
      | share_type             | user           |
      | file_source            | A_NUMBER       |
      | path                   | /simple-folder |
      | permissions            | read,share     |
      | stime                  | A_NUMBER       |
      | storage                | A_NUMBER       |
      | mail_send              | 0              |
      | uid_owner              | user0          |
      | file_parent            | A_NUMBER       |
      | displayname_owner      | User Zero      |
      | share_with             | user1          |
      | share_with_displayname | User One       |
    And the additional sharing attributes for the response should include                                                                              # FeatureContext::checkingAttributesInLastShareResponse()
      | scope         | key       | enabled |
      | permissions   | download  | false   |
      | richdocuments | watermark | true    |
      | richdocuments | print     | true    |
      Failed asserting that an array has the subset Array &0 (
          0 => Array &1 (
              'scope' => 'permissions'
              'key' => 'download'
              'enabled' => false
          )
          1 => Array &2 (
              'scope' => 'richdocuments'
              'key' => 'watermark'
              'enabled' => true
          )
          2 => Array &3 (
              'scope' => 'richdocuments'
              'key' => 'print'
              'enabled' => true
          )
      ).
      --- Expected
      +++ Actual
      @@ @@
       array (
         0 => 
         array (
      -    'scope' => 'permissions',
      -    'key' => 'download',
      -    'enabled' => false,
      -  ),
      -  1 => 
      -  array (
           'scope' => 'richdocuments',
           'key' => 'watermark',
           'enabled' => true,
         ),
      -  2 => 
      +  1 =>
    And the downloaded content when downloading file "simple-folder/lorem.txt" for user "user0" with range "bytes=0-17" should be "This is lorem text" # FeatureContext::downloadedContentWhenDownloadingForUserWithRangeShouldBe()
    And as "user1" file "simple-folder/lorem.txt" should exist                                                                                         # FeatureContext::asFileOrFolderShouldExist()
    But the downloading of file "simple-folder/lorem.txt" for user "user1" should fail with error message                                              # FeatureContext::userDownloadsFailWithMessage()
      """
      Access to this resource has been denied because it is in view-only mode.
      """
SCENARIO RESULT: (fail)

--- Failed scenarios:

    /home/phil/git/owncloud/core/apps-external/richdocuments/tests/acceptance/features/webUISecureView/main.feature:51

1 scenario (1 failed)
12 steps (8 passed, 1 failed, 3 skipped)
0m33.39s (25.04Mb)
```

If I change

**Comment by phil-davis** [comment]
If I change `$value` to `true` at https://github.com/owncloud/core/pull/36024/commits/2ef190a4fce0d0ab6173da1304ffaa35e95ede95#r317479629 then the tests pass.

@DeepDiver1975 ?

**Comment by phil-davis** [comment]
Fixed by #36086
